triggers.lesson14/styles-for-triggers.css

```diff
@@ -0,0 +1,142 @@
+.container div {
+    display: flex;
+    margin: 150px;
+    padding: 10px 150px 10px 10px;
+    border: rgb(7, 69, 151) solid 2px;
+}
+
+.content div {
+    font-size: 20px;
+    padding: 0px;
+    border: 2px solid rgb(201, 63, 139);
+    margin: 10px;
+}
+
+.container {
+    position: absolute;
+    animation: slide-from-left 1s;
+    animation-duration: 3s;
+    animation-timing-function: ease-out;
+}
+
+@keyframes slide-from-left {
+    from {
+        left: -1000px;
+    }
+
+    to {
+        left: 0;
+    }
+}
+
+.tooltip {
+    position: relative;
+    display: inline-block;
+    cursor: default;
+}
+
+.tooltip span {
+    display: inline-block;
+    width: 150px;
+    position: absolute;
+    border-radius: 5px;
+    background-color: silver;
+    padding: 5px;
+    text-align: center;
+    visibility: hidden;
+    opacity: 0;
+    transition: opacity 1s, visibility 1s;
+}
+
+.tooltip-top {
+    bottom: 160%;
+    left: -40%;
+}
+
+.tooltip:hover .tooltip-top {
+    opacity: 1;
+    visibility: visible;
+}
+
+.tooltip-top::before {
```

Comment:
Same optimization you can do with arrows. 
Use "after" for all of them to make the arrows. And write just once all common styles for arrows. And only set custom border for each direction.